applications/vessel-history/src/features/profile/components/activity/Activity.tsx

```diff
@@ -0,0 +1,102 @@
+import { ActivityEvent, ActivityEventGroup } from 'types/activity'
+import I18nDate from 'features/i18n/i18nDate'
+import styles from './Activity.module.css'
+import ActivityDate from './ActivityDate'
+
+interface InfoProps {
+  vessel: VesselWithHistory | null
+  lastPosition: any
+  lastPortVisit: any
+}
+
+const Activity: React.FC<InfoProps> = (props): React.ReactElement => {
+  const { t } = useTranslation()
+  const vesselId = useSelector(selectVesselId)
+
+ 
+  const dispatch = useDispatch()
+  useEffect(() => {
+    if (vesselId) {
+      console.log(vesselId)
+      dispatch(fetchVesselActivityThunk({vesselId}))
+    }
+  }, [dispatch, vesselId])
+
+  const onOpenGroup = useCallback((index: number) => {
+    dispatch(toggleGroup({index}))
+  },[dispatch])
+
+  const eventGroups = useSelector(selectVesselActivity)
+  console.log(eventGroups)
+  return (
+    <Fragment>
+      <div className={styles.activityContainer}>
+        {eventGroups && eventGroups.map((group: ActivityEventGroup, groupIndex) => ( 
+          <Fragment key={groupIndex}>
+            {!group.open && group.entries && ( 
+              <Fragment>
+                  <div className={styles.event} >
+                    <div className={styles.eventIcon}>
+                      <i></i>
+                      <span className={styles.eventCount}>{group.entries.length}</span>
+                    </div>
+                    <div className={styles.eventData}>
+                      <div className={styles.date}>
+                        <I18nDate date={group.entries[0].start} format={DateTime.DATE_SHORT} /> - 
+                        <I18nDate date={group.entries[group.entries.length - 1].end} format={DateTime.DATE_SHORT} />
+                      </div>
+                      <div className={styles.description}>
+                      {group.entries.length} Fishing events in {group.event_place}
+                      </div>
+                      
+                    </div>
+                    {group.entries.length > 1 && ( 
+                      <div className={styles.actions}>
+                        <IconButton icon="compare" size="small" onClick={() => onOpenGroup(groupIndex)}></IconButton>
+                      </div>
+                    )}
+                  </div>
+                <div className={styles.divider}></div>
+              </Fragment>
+            )}
+            {group.open && group.entries && group.entries.map((event: ActivityEvent, eventIndex) => ( 
+              <Fragment key={eventIndex}>
+                  <div className={styles.event} >
+                    <div  className={styles.eventIcon}>
+                      <i></i>
+                    </div>
+                    <div className={styles.eventData}>
+                      <ActivityDate event={event}/>
+                      <div className={styles.description}>
+                        Fishing in ???? {event.start}
```

Comment:
Missing translation here. What are the `????` ? Is any data that you need to display there missing?

Reply:
yes, is not clear what is the data to display from the api